### DISCUSSION TITLE
Improvement in snap

### DISCUSSION THREAD

**Discussion by soumyaDghosh** [hwittenborn/celeste#99] [Ideas]
Recently, I tried my `just` based app, and it worked completely fine without the `libadwaita-1`. It's automatically used from the gnome-content snap. But I still can't figure it out, how did you find these values?
```
  - type: apt
    components: [main, multiverse, restricted, universe]
    suites: [kinetic, kinetic-backports, kinetic-security, kinetic-updates]
    key-id: F6ECB3762474EDA9D21B7022871920D1991BC93C
    url: http://security.ubuntu.com/ubuntu
```

Can you help me about these please?

**Comment by Fyafir** [comment]
Need to create a step-by-step guide on how to sync each Cloud.
